Iguana2024/devops-2024
# Added provision.sh for setup configuration of SFTP servers on vm's

## What type of PR is this? (check all applicable)

- [✓] Optimization

## Description
Created a provision.sh that has a configuration for SFTP server, as well as minor configs for other resources aimed to run necessary scripts for cronjobs as well as main cofig for SFTP
 ## How to validate
 Set the IP address passed as a command-line argument
 Created Log start of operation
 Install vsftpd and rkhunter
 Install OpenSSH Server
 Configure SSHD for SFTP
 After that we configure SSH/SFTP
 Finally update, prepare, and check rkhunter
 All the steps are listed in the script itself
 Generally you should go with vagrant up and the output will display if everything configured and running.
